pkgs/games/srb2kart/default.nix

```diff
@@ -0,0 +1,128 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchzip
+, cmake
+, pkg-config
+, nasm
+, makeDesktopItem
+, copyDesktopItems
+, SDL2
+, SDL2_mixer
+, zlib
+, libopenmpt
+, libgme
+, libpng
+, curl
+}:
+
+let
+  version = "1.3";
+
+  assets = fetchzip {
+    name = "srb2kart-assets";
+    url = "https://github.com/STJr/Kart-Public/releases/download/v${version}/srb2kart-v${lib.replaceStrings ["."] [""] version}-Installer.exe";
+    sha256 = "sha256-WDE7noJTy+JLsSh5ETcTFe1b4jz2RgZ/zuYFkQ2U+w8=";
+    extension = "zip";
+    stripRoot = false;
+    extraPostFetch = ''
+      rm $out/*.dll $out/*.exe $out/*.bat
+    '';
+  };
+
+  description = "A kart racing mod based on the 3D Sonic the Hedgehog fangame Sonic Robo Blast 2";
+  desktopItem = makeDesktopItem {
```

Comment:
We could inline both of them and reference meta.description.

Reply:
Done